.github/workflows/ci.yml

```
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22.13.1
```

Hey, let's not pin down the minor & patch version, so that we don't have to update this file every time a new Node.js version is released.

```suggestion
          node-version: 22
```

(The same comment applies to other places below, too.)